test/cpp/jit/test_misc.h

```diff
@@ -420,11 +421,11 @@ void testCustomFusionNestedBlocks() {
       g,
       [](Node* n) { return n->kind() == aten::mul; },
       Symbol::fromQualString("prim::FusionGroup"));
-  
+
   // Could be done in more efficient ways, but this is only a test.
   std::function<bool(const Block*, Symbol)> dfs = [&](const Block* b, Symbol s) {
       for (auto node : b->nodes()) {
-          if (node->kind() == s) 
+          if (node->kind() == s)
```

Comment:
I don't know if you'd like to fix it in this PR or nor, but the if-body should be inside {} (I think linker might yell at you because of that now).